common/dist/scripts/habitrpg-shared.js

```diff
@@ -2772,12 +2776,16 @@ api.spells = {
       target: 'task',
       notes: t('spellRogueBackStabNotes'),
       cast: function(user, target) {
-        var bonus, _crit;
+        var bonus, req, _crit;
         _crit = user.fns.crit('str', .3);
         target.value += _crit * .03;
         bonus = (target.value < 0 ? 1 : target.value + 1) * _crit;
         user.stats.exp += bonus;
-        return user.stats.gp += bonus;
+        user.stats.gp += bonus;
+        req = {
+          language: user.preferences.language
+        };
+        return user.fns.updateStats(user.stats, req);
```

Comment:
Yep, makes sense :) `npm start` and `npm test` already run `grunt run:dev`, so there shouldn't be any use cases where people will try to do stuff without this file being compiled. :+1: